src/storage/activities.tsx

```diff
@@ -160,9 +188,33 @@ export const useActivities = (): [Activities, ModifyActivities] => {
     return false;
   };
 
+  /**
+   * Removes an entry from the specified activity list.
+   * 
+   * @remarks: This function ignores all matching instances and does not check
+   *  if they connect or not.
+   *
+   * @param entry - The entry to be removed
+   * @returns A new activity list with the entry removed
+   */
+  const remove = (activity: ActivitiesEntry, day: ActivitiesDay, activities: Activities): Activities => {
+    const dayIndex = activities.findIndex(elem => elem === day);
```

Comment:
This should use activityDayEq

Reply:
Why was this marked as resolved? @malin-mallan